stubs/taint/core_privacy_security/server_side_template_injection_sinks.pysa

```diff
@@ -0,0 +1,29 @@
+# SSTI
+def jinja2.environment.Template.__new__(cls, source: TaintSink[ServerSideTemplateInjection]): ...
+def jinja2.environment.Environment.from_string(self, source: TaintSink[ServerSideTemplateInjection, ViaTypeOf[self]]): ...
+def jinja2.environment.Environment.compile_expression(self, source: TaintSink[ServerSideTemplateInjection, ViaTypeOf[self]]): ...
+
+#django.template
+def django.template.engine.Engine.from_string(self, source: TaintSink[ServerSideTemplateInjection]): ...
+def django.template.base.Template.__init__(self, source: TaintSink[ServerSideTemplateInjection], **kwargs): ...
+
+#mako
+def mako.template.Template.__init__(self, text: TaintSink[ServerSideTemplateInjection]): ...
+
+#chameleon
+def chameleon.template.BaseTemplate.__init__(self, body: TaintSink[ServerSideTemplateInjection], **kwargs): ...
```

Comment:
Can we just have the tainted parameters only defined and skip the rest? Does it work?
ie, instead of,
`def chameleon.template.BaseTemplate.__init__(self, body: TaintSink[ServerSideTemplateInjection], **kwargs): ...`
Can we have
`def chameleon.template.BaseTemplate.__init__(body: TaintSink[ServerSideTemplateInjection]): ...`

Asking cause these extra parameters are actually not required by Pysa, and they make the model harder to maintain, so can we prefer to not include them?

(Applies to all functions in the file)